src/coordinator/services/m3coordinator/server/server.go

```diff
@@ -159,6 +174,9 @@ func Run(runOpts RunOptions) {
 		}
 	}
 
+	namespaces := clusters.ClusterNamespaces()
+	logger.Info("resolved cluster namespaces", zap.Int("namespaces", len(namespaces)))
```

Comment:
Would it be useful to list the names instead of the count?

Reply:
Sure thing.